### ISSUE TITLE
Metric

### ISSUE THREAD

**Issue by sucessfullys** [JinyuanLiu-CV/SegMiF#29]
Thank you very much for your excellent work, but I have a doubt. When I run the pre-trained weights on the FMB dataset, the metrics are much higher than those in the paper, but when I reproduce it myself, the metrics are slightly lower than those in the paper. The test code is the same. Thank you very much for your response.

**Comment by Cup-Laboratory** [comment]
Hello, may I ask if you have been able to run this experimental code successfully? I'm a beginner and my code doesn't work. Could you please help me run the code once?

**Comment by sucessfullys** [comment]
Once the path is set correctly, it should work. Specifically, what issue are you encountering?







-----原始邮件-----
发件人:Cup-Laboratory ***@***.***>
发送时间:2025-09-04 18:33:58 (星期四)
收件人: JinyuanLiu-CV/SegMiF ***@***.***>
抄送: "Yufa Duan" ***@***.***>, Author ***@***.***>
主题: Re: [JinyuanLiu-CV/SegMiF] Metric (Issue #29)


Cup-Laboratory left a comment (JinyuanLiu-CV/SegMiF#29)

Hello, may I ask if you have been able to run this experimental code successfully? I'm a beginner and my code doesn't work. Could you please help me run the code once?

**Comment by xiaoyaozhengzheng** [comment]
Hello, after downloading the code we did the path configuration, but there are still many problems. The project takes 30 hours to run, how long do you need when you run the project? The results are a bit unsatisfactory, and there are warnings about some channel mismatches.
